.gitignore

```diff
@@ -3,3 +3,9 @@ node_modules/
 dist/
 .env*
 .vscode
+.DS_STORE
+.idea
+**/*.js
```

Comment:
I don't think we should ignore js files. Is there a reason to do that?

Reply:
I guess there isnt, if the tsc is being used in the right context

Reply:
removed